conf/checksums

```diff
@@ -1,3 +1,4 @@
+accumulo:2.0.0:df172111698c7a73aa031de09bd5589263a6b824482fbb9b4f0440a16602ed47
```

Comment:
This hash looks shorter than the others... maybe you need sha512?

Reply:
changed to sha512